workerapi/server/setup.go

```diff
@@ -59,10 +58,6 @@ func Defaults() (*ice.MagicBag, jsonconfig.Schema) {
 
 		func() (*temp.TempDir, error) { return temp.TempDirDefault() },
 
```

Comment:
Because we need to override it in Twitter (and right now there's no way to override a default in ice)